### ISSUE TITLE
Fix colour contrast issue on Privacy Policy Page in Dark mode

### ISSUE THREAD

**Issue by StephanosNikitis** [Nikhilrsingh/car-transport-service#893]
### Description
Text colour contrast doesn't match in dark mode due to which the text is not visible on Privacy Policy Page.

### Solution
Fix colour contrast on dark mode so that text can be visible.

### Screenshot

<img width="1899" height="961" alt="Image" src="https://github.com/user-attachments/assets/e9519e31-70f3-4f29-9a53-ecf8470425cf" />

<img width="1898" height="959" alt="Image" src="https://github.com/user-attachments/assets/955b755a-2d5e-41e7-8798-b4985e6d4f0f" />

---

Please assign me this issue under SWOC'26.

**Comment by Nikhilrsingh** [comment]
<img width="1710" height="1107" alt="Image" src="https://github.com/user-attachments/assets/f368ce9d-bad5-4d70-9a1e-bcdfa3e1a9af" />

@StephanosNikitis please check from live server not from GitHub link it is already solved
